releasenotes/notes/2.0/fix-cached-params-update-4d2814b698fa76b4.yaml

```diff
@@ -1,8 +1,8 @@
 ---
 fixes:
   - |
-    Fixed a bug in :meth:`.QuantumCircuit.assign_parameters`, occurring when assigning parameters
-    to standard gates whose definition has already been triggered. In this case, the new values
+    Fixed a bug in :meth:.QuantumCircuit.assign_parameters, which occurred when assigning parameters 
```

Comment:
We can delete this release note it was already released in 1.3.0: https://github.com/Qiskit/qiskit/pull/13495